docs/UserGuide.md

```diff
@@ -534,8 +602,17 @@ With EduMate's `untag` command, you can remove groups or modules from your exist
 </div>
 
 What you should see:<br>
-{GUI}<br>
-{Explanation}
+
+<img src="images/ug_cmds_pics/untag_cmd.png" style="width:80%;margin:0 10%">
+
+<center>Diagram 13: Amirah after removing the CS2109S Module Tag.</center>
+
+<br>
+
+##### Explanation:
+Say you realise that Amirah accidentally told you the wrong information, you can remove the CS2109S module tag
+with the `untag 1 m/CS2109S MONDAY  16 18` command. The tag will disappear from Amirah's profile in the Profile Panel. 
```

Comment:
The cs2109s came a little abruptly, maybe can signal at the front that that is the wrong information.
e.g Say you realise that Amirah does not actually take CS2109S, you can easily remove the CS2109S module tag
with the `untag 1 m/CS2109S MONDAY  16 18` command.